util/include/evp_hash.hpp

```diff
@@ -97,6 +98,12 @@ class EVPHash {
     updating_ = false;
     return digest;
   }
+  static std::string toHexString(const Digest& digest) {
```

Comment:
This seems like something that can be used in many places throughout the code.
Consider making it a template so that every collection of bytes (vector for instance) can be converted to a hex string as well.

Reply:
You are right, we have it in a Sliver, Digest and  STDigest classes as well